src/utils/page.js

```diff
@@ -38,6 +41,37 @@ export function getPage (win = window) {
   return detectedPageUrl
 }
 
+/**
+ * @return {string|undefined}
+ */
+export function getContextElements (contextSelectors, contextElementsLength, win = window) {
```

Comment:
Why is `win` needed here? I saw that we are doing it in other functions functions also and the generated code does some magic like `var win = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : window;` But why?